osbenchmark/benchmark.py

```diff
@@ -205,6 +213,7 @@ def add_workload_source(subparser):
         default=metrics.GlobalStatsCalculator.DEFAULT_LATENCY_PERCENTILES)
     compare_parser.add_argument(
         "--results-format",
+        "-rfo",
```

Comment:
For results format and results file, I think we can just stick with the longer names and not assign them short-hand flags.